components/style/values/computed/image.rs

```diff
@@ -454,3 +455,73 @@ impl ToComputedValue for specified::LengthOrPercentageOrKeyword {
         }
     }
 }
+
+#[derive(Clone, Debug, PartialEq)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+pub enum AngleOrCorner {
```

Comment:
This type (and its `impl`) should be moved to `components/style/values/computed/mod.rs`